lib/internal/Magento/Framework/MessageQueue/Topology/Config/QueueConfigItem/DataMapper.php

```diff
@@ -59,21 +62,28 @@ public function __construct(
      * Get mapped config data.
      *
      * @return array
+     * @throws LocalizedException
      */
-    public function getMappedData()
+    public function getMappedData(): array
     {
         if (null === $this->mappedData) {
             $this->mappedData = [];
             foreach ($this->configData->get() as $exchange) {
                 $connection = $exchange['connection'];
                 foreach ($exchange['bindings'] as $binding) {
                     if ($binding['destinationType'] === 'queue') {
-                        $queueItems = $this->createQueueItems($binding['destination'], $binding['topic'], $connection);
-                        $this->mappedData = array_merge($this->mappedData, $queueItems);
+                        $queueItems = $this->createQueueItems(
+                            (string) $binding['destination'],
```

Comment:
1. Could you remove the space between type casting and variable in all places in order to have similar code formatting as in other places in Magento, like this?
```suggestion
                            (string)$binding['destination'],
```

2. Do we really need this casting? Do we have any case where we'll have another type?